### PR TITLE
Pf/gl 2

### PR DESCRIPTION
**Pull request by pford68**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,6 +16,9 @@
 
 	<properties>
 		<java.version>1.8</java.version>
```
Comment (ecumali) [inline pom.xml]: 11

File: src/main/java/com/giantmachines/biblio/serializers/ReviewSerializer.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.giantmachines.biblio.model.Review;
+import com.giantmachines.biblio.security.CurrentUser;
+
+import java.io.IOException;
+
+public class ReviewSerializer extends JsonSerializer<Review> {
```
Comment (ecumali) [inline src/main/java/com/giantmachines/biblio/serializers/ReviewSerializer.java]: You should get this out of the box with Jackson

File: src/main/resources/application-production.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+spring.profiles.active=production
+
+# Hibernate
+spring.jpa.hibernate.ddl-auto=update
```
Comment (ecumali) [inline src/main/resources/application-production.properties]: I wouldn't use "update" in prod

Comment (pford68) [reply src/main/resources/application-production.properties]: Ah, mistake

File: pom.xml [hunk 4]
```diff
@@ -43,13 +46,20 @@
...
 			<scope>runtime</scope>
-			<version>1.4.199</version>
+			<version>${h2.version}</version>
 		</dependency>
 
 		<dependency>
 			<groupId>mysql</groupId>
 			<artifactId>mysql-connector-java</artifactId>
-			<version>8.0.17</version>
+			<version>${mysql.version}</version>
```
Comment (ecumali) [inline pom.xml]: Any reason you are using mysql? I think the module lists postgres

**Pull request merged**